src/routes/login/login.js

```diff
@@ -3,12 +3,28 @@ import PropTypes from 'prop-types';
 import { Grid } from 'semantic-ui-react';
 import { LoginForm } from 'components/forms';
 import { authenticationService } from 'api';
-import { SubmissionError } from 'redux-form';
+
+function getErrorMessage(response) {
```

Comment:
move this to a utility function, since it seems to be used in multiple places. and check if the 404 is actually supposed to happen.

Reply:
After I simplified the error message handling. There is no code shared so I have not moved it to a utility function. I do it if I later discover that there is shared code.